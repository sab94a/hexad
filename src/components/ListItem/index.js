import React from 'react';

const ListItem = ({ title }) => (<div>{ title }</div>);

export default React.memo(ListItem);
