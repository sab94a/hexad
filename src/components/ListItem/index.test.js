import React from 'react';
import { shallow } from 'enzyme';
import { shallowToJson } from 'enzyme-to-json';
import ListItem from './'

describe('ListItem', () => {
    const TITLE = 'TITLE';
    let component;

    it('Should render correctly', () => {
        component = shallow(<ListItem title={ TITLE }/>);
        expect(shallowToJson(component)).toMatchSnapshot();
    });

    it('Should render correct title', () => {
        const title = component.text();
        expect(title).toEqual(TITLE);
    })
})
