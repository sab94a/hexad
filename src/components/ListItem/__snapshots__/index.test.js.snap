// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ListItem Should render correctly 1`] = `
<div>
  TITLE
</div>
`;
